runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -90,6 +90,28 @@
     @Getter
     private boolean logHasNoCheckPoint = false;
 
+    private boolean whiteList = false;
+    private List<UUID> streamsToLoad = new ArrayList<>();
+
+    /**
+     * Enable whiteList mode where we only reconstruct
+     * the streams provided through this api.
+     * @param streamsToLoad
+     */
+    public FastObjectLoader addStreamsToLoad(List<UUID> streamsToLoad) {
+        whiteList = true;
+        this.streamsToLoad.addAll(streamsToLoad);
+
+        // Generate the streamsCheckpointId (we need to allow them as well)
+        streamsToLoad.forEach((streamId) ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "streamId" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)